roseus/euslisp/roseus-utils.l

```diff
@@ -1039,10 +1039,11 @@
 ;; misc function
 ;;
 (ros::roseus-add-srvs "std_srvs")
-(defun call-empty-service (srvname &key (wait nil))
-  "Call empty service"
-  (if wait (ros::wait-for-service srvname))
-  (ros::service-call srvname (instance std_srvs::EmptyRequest :init)))
+(defun call-empty-service (srvname &key (wait nil) (timeout -1)) ;; If timeout is -1 (default), waits until the node is shutdown. If timeout is 0, returns immediately.
```

Comment:
@Affonso-Gui 
`, waits until the node is shutdown` is this correct?
`, waits until the service becomes available` ?

Reply:
Waits until either the service becomes available or the node is shutdown.

This part is directly from the rosscpp documentation, so I don't think this is a problem.
```
If timeout is -1 (default), waits until the node is shutdown
```

http://docs.ros.org/en/noetic/api/roscpp/html/namespaceros_1_1service.html#aabe996581255345b3383e66eaaedef5a

In a separate note, timeout=-1 will NOT respond to C-c, which can lead to disappointing results in the interpreter.